### PR TITLE
Releases: Rename "Modules" to "NODE_MODULE_VERSION"

### PR DESCRIPTION
**Pull request by fhemberger**
Clarifies "NODE_MODULE_VERSION", see #419 and https://github.com/nodejs/build/issues/219.

/cc @charleswhchan @rvagg

### CODE REVIEW

File: locale/en/download/releases.md [hunk 1]
```diff
@@ -3,5 +3,5 @@ layout: download-releases.hbs
 title: Previous Releases
 iojs:
   intro: "Releases 1.x through 3.x were called \"io.js\" as they were part of the io.js fork. As of Node.js 4.0.0 the former release lines of io.js converged with Node.js 0.12.x into unified Node.js releases."
-modules: "\"Modules\" refers to the ABI (application binary interface) version number of Node.js, used to determine which versions of Node.js compiled C++ add-on binaries can be loaded in to without needing to be re-compiled. This version number is also referred to as <code>NODE_MODULE_VERSION</code>."
+modules: "<code>NODE_MODULE_VERSION</code> refers to the ABI (application binary interface) version number of Node.js, used to determine which versions of Node.js compiled C++ add-on binaries can be loaded in to without needing to be re-compiled. It used to be stored as hex value in earlier version, but is now represented as an Integer."
```
Comment (rvagg) [inline locale/en/download/releases.md]: probably doesn't need to be "Integer" with the uppercase "I".

**Comment by rvagg** [comment]
also /cc @maxogden

**Comment by Knighton910** [comment]
hex value in earlier version, but is now represented as an Integer.
:large_blue_circle:  I hate to nitpick but this should be versions or  in an earlier version.

**Comment by fhemberger** [comment]
@kelthenoble You're right. Fixed.

**Comment by Knighton910** [comment]
:+1: Sweet :100:

**Comment by max-mapper** [comment]
I had it labeled as NODE_MODULE_VERSION in the original pull request I made, the feedback was that it was too long. But I am personally +1 on it being longer so that its more clear to the uninitiated.

**Comment by fhemberger** [comment]
Merged, thanks everyone.

**Pull request merged**